Formula/orgalorg.rb

```diff
@@ -0,0 +1,25 @@
+class Orgalorg < Formula
+  desc "Parallel SSH commands executioner and file synchronization tool"
+  homepage "https://github.com/reconquest/orgalorg"
+  url "https://github.com/reconquest/orgalorg.git",
+      tag:      "1.0.1",
+      revision: "7ccd02c521f79e04720aa929e21715d46668162f"
+  license "MIT"
+  head "https://github.com/reconquest/orgalorg.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-mod=mod", "-ldflags", "-s -w -X main.version=#{version}", *std_go_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/orgalorg --version")
+    assert_match "orgalorg - files synchronization on many hosts.", shell_output("#{bin}/orgalorg --help")
+
+    output = shell_output("#{bin}/orgalorg -u tester -o 127.0.0.1:9999 -C uptime 2>&1", 1)
+    assert_match(/connecting to cluster failed/, output)
+    assert_match(/dial tcp 127.0.0.1:9999: connect: connection refused/, output)
+    assert_match(/can't connect to address: \[tester@127.0.0.1:9999\]/, output)
```

Comment:
I think with this we'll be a little more certain the test will always fail. But the test is a lot better like this.
```suggestion
    port = free_port
    output = shell_output("#{bin}/orgalorg -u tester -o 127.0.0.1:#{port} -C uptime 2>&1", 1)
    assert_match("connecting to cluster failed", output)
    assert_match("dial tcp 127.0.0.1:#{port}: connect: connection refused", output)
    assert_match("can't connect to address: [tester@127.0.0.1:#{port}]", output)
```

Reply:
I'm not familiar with testing in Ruby could you tell me please where is from the "free_port" function? I tried to figure out the ways of testing and homebrew cookbook unfortunately didn't gave me a lot, so I've been forced to read brew formulas from repo.
Ok, I've added suggestion, thanks.
I think it would be better to point new people to good doc, so they can write good tests before PR for sure.

Reply:
> I'm not familiar with testing in Ruby could you tell me please where is from the "free_port" function?

It's a brew specific function to find an available port.

> so I've been forced to read brew formulas from repo.

This is the best way since homebrew maintainers are developers and not writers. This is the place that is guaranteed to have the latest features.

>  I think it would be better to point new people to good doc, so they can write good tests before PR for sure.

If you have any suggestions to write a good doc I'd happily review a pull request for it.

Reply:
> It's a brew specific function to find an available port.
Is there any place where i could get read about them all?

> If you have any suggestions to write a good doc I'd happily review a pull request for it.
Would keep in mind that, thanks

Reply:
> Is there any place where i could get read about them all?

Not really a place where you can find things to use in a formulae, but the brew api docs are here:
https://rubydoc.brew.sh/Homebrew/FreePort.html

Reply:
thanks